data/src/main/java/hera/store/unit/ModuleSettingsAccessUnit.java

```diff
@@ -23,4 +26,14 @@ public ModuleSettingsAccessUnit() {
 	public List<ModuleSettings> forModule(Long guild, int command) {
 		return data.stream().filter((m) -> m.getGuild().equals(guild) && m.getCommand() == command).collect(Collectors.toList());
 	}
+
+	public void update(ModuleSettings ms) {
+		try {
+			dao.update(ModuleSettingsPO.class, ms);
```

Comment:
Here you update the database, but you also need to update the store. There is a field `data` in StorageAccessUnit which contains all the modulesettings initialy loaded from the db. Update the module setting in there too, else we would have to reload from the DB again

Reply:
not needed cuz of reference

Reply:
yes you're right, I overlooked that. Good catch!